src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -163,13 +161,13 @@ private void processBlob(String blobName, String containerName) {
             ZipEntry entry = null;
 
             while ((entry = zipStream.getNextEntry()) != null) {
-                if (ENVELOPE_ENTRY_NAME.equals(entry.getName())) {
+                if (ZipVerifiers.DOCUMENTS_ZIP.equals(entry.getName())) {
```

Comment:
`ZipVerifiers.ENVELOPES_ZIP` may be ?

Reply:
For now I'm just removing the duplication of contants.

The other name is not great either, currently there are those constants in `ZipVerifiers` class:

```java
    public static final String DOCUMENTS_ZIP = "envelope.zip";
    public static final String SIGNATURE_SIG = "signature";
    public static final String INVALID_SIGNATURE_MESSAGE = "Zip signature failed verification";
```

I'd probably move the file names constants to a separate class (like `ZipFileEntries`) and call them `ENVELOPE_ZIP` and `SIGNATURE` respectively.